icu4c/source/test/intltest/dtfmttst.cpp

```diff
@@ -3478,20 +3478,28 @@ void DateFormatTest::TestTimeZoneInLocale()
         UnicodeString actual;
 
         SimpleDateFormat smptfmt("Z", locale, status);
+        ASSERT_OK(status);
         assertEquals("TimeZone from SimpleDateFormat constructor",
                      expectedTimezone, smptfmt.getTimeZone().getID(actual));
         assertEquals("Calendar from SimpleDateFormat constructor",
                      testLine[2], smptfmt.getCalendar()->getType());
 
         LocalPointer<DateFormat> datefmt(
                 DateFormat::createDateInstance(DateFormat::kDefault, locale));
+        if (datefmt == nullptr){
```

Comment:
Please use a space before `{`, here and below.

Reply:
Done.